roomies_project_plan.md

```
@@ -338,10 +432,11 @@ This file is used specifically to verify the application can connect to all Azur
 | `ACS_FROM_EMAIL` | Verified sender email on ACS |
-| `STORAGE_ADAPTER` | `azure` — activates `AzureBlobStorageAdapter` |
+| `STORAGE_ADAPTER` | `azure` — activates `AzureBlobAdapter` |
 | `NODE_ENV` | `production` |
 | `PORT` | `8080` (Azure App Service default) |
+| `ALLOWED_ORIGINS` | `https://roomies.in,https://www.roomies.in` |
 
 ---
 
-*— End of Roomies Project Plan —*
\ No newline at end of file
+*— Last updated: phase1/foundation merged —*
```

markdownlint-cli2 (0.21.0)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)